java/vector/src/main/codegen/data/ValueVectorTypes.tdd

```diff
@@ -56,8 +56,10 @@
         { class: "Int", valueHolder: "IntHolder"},
         { class: "UInt4", valueHolder: "UInt4Holder" },
         { class: "Float4", javaType: "float" , boxedType: "Float", fields: [{name: "value", type: "float"}]},
-        { class: "IntervalYear", javaType: "int", friendlyType: "Period" }
-        { class: "Time", javaType: "int", friendlyType: "DateTime" }
+        { class: "DateDay", javaType: "int", boxedType: "Integer", friendlyType: "DateTime" },
```

Comment:
@julienledem @jacques-n to be clear, I'm proposing committing as-is and fixing them to be {Date,Time,TimeStamp}{4,8} before releasing 0.3.0.  @icexelloss maybe you want to take that since you have a better idea of how to do it than I do?